NEWS.md

```diff
@@ -1,5 +1,11 @@
 # ggplot2 (development version)
 
+* `scale_*_manual()` no longer displays extra legend keys, or changes their 
+  order, when a named `values` argument has more items than the data. The 
+  previous behaviour can be replicated by using 
+  `scale_*_manual(values = vals, limits = names(vals))`. (@teunbrand, @banfai, 
+  #4511, #4534)
```

Comment:
The word "previous behaviour" might be a bit confusing to those who want further previous behaviour, i.e. behaviour before v3.3.4. How about describing it a bit more explicitly something like "To display all the `values` on the legend, use `scale_*_manual(values = vals, limits = names(vals))`"? What do you think?

Reply:
Agreed, "previous behavior" is confusing. Better to describe the specific behavior. I like the proposed sentence.